app/controllers/application_controller.rb

```diff
@@ -36,7 +36,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_templates
   helper_method :admin?, :logged_in?, :highlight_all
 
-  #  protect_from_forgery :secret => '112141be0ba20082c17b05c78c63f357'
+  protect_from_forgery with: :exception
```

Comment:
Before we enable this, application requires an auditing on which URL endpoints need an exception from this rule. (One that comes to mind is the action which received incoming emails via HTTP).